### PR TITLE
additional validation for access legacy operator

### PR DESCRIPTION
**Pull request by ionox0**
I also moved the access tests to their own subfolder to keep it a bit more organized

### CODE REVIEW

**Comment by ionox0** [comment]
WIP I probably want to add some tests

File: runner/operator/access/v1_0_0/legacy/__init__.py [hunk 1]
```diff
@@ -38,7 +55,18 @@
...
 """
 def get_missing_fields(source, fields):
-    return list(filter(lambda field: field not in source or not source[field], fields))
+
+    def validate(field):
+        if field not in source:
+            return True
+        elif not source[field]:
+            return True
+        elif type(source[field]) == list:
```
Comment (ionox0) [inline runner/operator/access/v1_0_0/legacy/__init__.py:64]: @aef- this is to handle the failure we had where just one of the fields in the `add_rg_PU: ['IDT234', None, ...]` field was null

**Comment by ionox0** [comment]
@aef- @sivkovic What does the `InputCreationFailedEvent` do? I was hoping that this would handle the notification for letting us know the inputs are invalid

**Comment by aef-** [comment]
I created this error message specifically for input validation
https://github.com/mskcc/beagle/blob/master/runner/operator/access/v1_0_0/legacy/__init__.py#L132

**Comment by ionox0** [comment]
Ok great, and that notifier is set up to put a comment in the Jira ticket right?

**Review by aef- — APPROVED** [review]

**Pull request merged**